README.md

```diff
@@ -8,149 +8,193 @@ Peter Rushforth
-  - [W3C/OGC Joint Workshop on Maps for the Web](#w3c-ogc-joint-workshop-on-maps-for-the-web)
 - [Considered Alternative Designs of MapML](#considered-alternative-designs-of-mapml)
-- [Alternative Approaches](#alternative-approaches)
 - [Stakeholder Feedback / Opposition](#stakeholder-feedback--opposition)
 - [References and Acknowledgements](#references-and-acknowledgements)
 
 <h2 id="the-problem">The Problem</h2>
 
-Web maps today are created using a wide range of technology stacks on both the client and server, some standard, some open, and some proprietary.  The complexity of choices and the wide variety of technologies required to create Web maps results in <a href="https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md">maps of highly variable usability and accessibility</a>.  This has in turn led to the creation of centralized mapping services, that may or may not be implemented using Web technology; in some cases, mapping services which work well on desktop Web browsers mostly bypass the mobile Web through creation of mobile platform mapping apps, where the ‘rules of the Web platform’ (such as device permissions) do not apply.  Some centralized mapping services, both on the Web but especially on mobile technology platforms, are constructed for the purpose of tracking the user’s location and their locations of (search) interest, and using that private location information to market and re-sell highly targeted advertising. 
-
-The problem to be solved, therefore, is to reduce the threshold complexity of creating accessible, usable and privacy-preserving Web maps, and to enable full use of Web platform standards such as HTML, URL, SVG, CSS and JavaScript in map creation, styling, presentation and interaction. 
+Web maps are <span style="text-decoration:underline;">not interoperable</span>. 
+For example, users cannot “mash up” a Leaflet map and an OpenLayers map, two of 
+the most popular open source Web mapping frameworks, even if they show the same 
+exact data source in the exact same location.  (The situation is even more 
+non-interoperable between open map data and privately managed map databases, 
+where the sole client may be that provided by the owner of the map data. The 
+lack of legal interoperability of map data is outside the scope of standardization.)
+
+The lack of Web map interoperability affects developers. The skill set and 
+vocabulary required to create a robust map is different from one mapping framework 
+to the next. They can be complex to <span style="text-decoration:underline;">create</span>, 
+with visual styling of map information being a discipline that is generally 
+outside the realm of Web technology, and different from one mapping framework to 
+the next.
+
+Web maps are [typically not accessible](https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md). 
+The knowledge required to <span style="text-decoration:underline;">use</span> Web 
```

Comment:
Same as https://github.com/Maps4HTML/MapML-Proposal/pull/56#pullrequestreview-1021619026, use `<ins>`.

Did you also consider the italic font-style?